### ISSUE TITLE
Add DateTime Selection 

### ISSUE THREAD

**Issue by BradleyLamitie** [HarshaKota/MSCS-710L-Project#41]
Should we make it so that the user can define their own session?

**Comment by HarshaKota** [comment]
Yes, but we need to figure out a way to handle the issue where if the user enters a session start time that falls in our dark zone ( period where the app wasn't open and data was not collected).

This will be added as part of an enhancement in the future.
